src/Razor/Microsoft.AspNetCore.Razor.Language/src/RazorCodeGenerationOptions.cs

```diff
@@ -133,6 +135,14 @@ public static RazorCodeGenerationOptions CreateDesignTime(Action<RazorCodeGenera
         /// </summary>
         public virtual bool OmitMinimizedComponentAttributeValues { get; }
 
+        /// <summary>
+        /// Gets a value that determines if localized component names are to be supported.
```

Comment:
```suggestion
        /// Gets or sets a value that determines if localized component names are to be supported.
```

Reply:
Will update when this goes into main, want to avoid blocking on this as it'll require another round through the CI.